### ISSUE TITLE
Connecting to NextCloud Talk fails

### ISSUE THREAD

**Issue by gkachru** [42wim/matterbridge#1177]
**Describe the bug**
I tried using the new ncTalk bridge using NextCloud 19.0.1 and the Talk 9 app. It fails with "unsupported protocol scheme".

**To Reproduce**
Setup a nctalk bridge in the config with Server, Login and Password
Start matterbridge 1.18.0

**Expected behavior**
nctalk connection should succeed

**Screenshots/debug logs**
```
level=error msg="Cannot Connect" prefix=nctalk
level=error msg="Bridge nctalk.demo failed to start: Get "/ocs/v2.php/cloud/capabilities": unsupported protocol scheme """ prefix=router
```
**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - Matterbridge version: 1.18.0 e3d8fe4f

**Additional context**
I've tested my settings against the capabilities end point using curl and it works fine. The following section is under ocs>data > capabilities
```
   <spreed>
    <features>
     <element>audio</element>
     <element>video</element>
     <element>chat-v2</element>
     <element>conversation-v2</element>
     <element>guest-signaling</element>
     <element>empty-group-room</element>
     <element>guest-display-names</element>
     <element>multi-room-users</element>
     <element>favorites</element>
     <element>last-room-activity</element>
     <element>no-ping</element>
     <element>system-messages</element>
     <element>mention-flag</element>
     <element>in-call-flags</element>
     <element>notification-levels</element>
     <element>invite-groups-and-mails</element>
     <element>locked-one-to-one-rooms</element>
     <element>read-only-rooms</element>
     <element>chat-read-marker</element>
     <element>webinary-lobby</element>
     <element>start-call-flag</element>
     <element>chat-replies</element>
     <element>circles-support</element>
     <element>force-mute</element>
     <element>chat-reference-id</element>
    </features>
    <config>
     <attachments>
      <allowed>1</allowed>
      <folder>/Talk</folder>
     </attachments>
     <chat>
      <max-length>32000</max-length>
     </chat>
     <conversations>
      <can-create>1</can-create>
     </conversations>
    </config>
   </spreed>
```

**Comment by gary-kim** [comment]
That's strange. I'll look into this

**Comment by gary-kim** [comment]
Just had a thought: in your config, did you put `https://` for your nctalk server url?

https://github.com/42wim/matterbridge/blob/e1629994bdbb4257fe91cff0b3fc665814c66ac8/matterbridge.toml.sample#L1392-L1395

**Comment by gkachru** [comment]
Yes the server URL does contain https. The config looks like below:
```
[nctalk.demo]
Server="https://xxxxx.xxxxx.com"
Login="nc-bot"
Password="xxxxx"
```

**Comment by gkachru** [comment]
I'm not familiar with golang but just glancing through the code perhaps the [RequestClient ](https://github.com/42wim/matterbridge/blob/e1629994bdbb4257fe91cff0b3fc665814c66ac8/vendor/gomod.garykim.dev/nc-talk/user/user.go#L76
) function needs to prefix t.NextcloudURL to the client.URL?

**Comment by gary-kim** [comment]
Oh, I'm not sure how I didn't find that. Fix incoming

**Comment by 42wim** [comment]
this is now available in v1.18.1
